spec/lib/regexp_preview/multi_line_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe RegexpPreview::MultiLine do
+  describe "#matches_json" do
+    subject { RegexpPreview::MultiLine.new(File.expand_path("./spec/support/fixtures/error0.log", Rails.root), "multiline", params).matches_json }
+
+    let :params do
+      tmp = {
```

Comment:
`tmp` variable name is a bit of fazzy, should be such as `request_parameters`, `req_params`, or just `params`